pages/post.module.css

```diff
@@ -53,6 +53,22 @@
   font-style: italic;
 }
 
+.pre_code_block {
+  background: rgb(15, 8, 28);
```

Comment:
Can this block re-use styles from the `.code` class? It already has background styles that work with dark mode and sets to font to a monospaced one.

Reply:
Yes, however, padding will change, it looks a little tight with the current code padding.